tests/settings.py

```diff
@@ -3,7 +3,7 @@
 
 TESTING = sys.argv[1:2] == ['test']
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
-
+SPATIALITE_LIBRARY_PATH = 'mod_spatialite.so'
```

Comment:
This doesn't belong here! 
There is a local settings file [here](https://github.com/openwisp/openwisp-controller/blob/master/tests/local_settings.example.py) you can use that for this setting! :smile: